services/galley/test/integration/API.hs

```diff
@@ -1229,6 +1248,53 @@ postMessageQualifiedLocalOwningBackendFailedToSendClients2 = do
       WS.assertMatch_ t wsBob (wsAssertOtr' encodedData convId aliceOwningDomain aliceClient bobClient encodedTextForBob)
       WS.assertMatch_ t wsChad (wsAssertOtr' encodedData convId aliceOwningDomain aliceClient chadClient encodedTextForChad)
 
+testBuildFailedToSend :: TestM ()
+testBuildFailedToSend = liftIO $ do
```

Comment:
The tests in here are unit tests, aren't they? Can you move them to a unit test module?